test/Runtime/FreeStanding/freestanding_only.c

```diff
@@ -0,0 +1,33 @@
+// RUN: %llvmgcc %s -emit-llvm -O0 -g -c  -D_FORTIFY_SOURCE=0 -o %t2.bc
```

Comment:
@jbuening  Here, I had to add `D_FORTIFY_SOURCE`.
It comes as a default option from the compiler/system and is enabled by default for MacOSX.
As far as I remember some Linux distributions show a similar behaviour.